<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyPointDialog</class>
 <widget class="QDialog" name="KeyPointDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="keypointcomboBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ISS关键点提取</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Harris3D关键点提取</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NARF关键点提取</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SIFT3D关键点提取</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>uniform sampling</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>voxel sampling</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="gridspinBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>3</number>
   </property>
   <property name="singleStep">
    <number>2</number>
   </property>
   <property name="value">
    <number>9</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="samplingspinBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="radiusspinBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="value">
    <number>15</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>半径</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>格网</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>采样</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>阈值</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ratiodoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>170</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
   <property name="value">
    <double>0.900000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>KeyPointDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>KeyPointDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
